File: Networking/Response/NewParticipantResponse.cs
using System.Text.Json.Serialization;
using mpp_proiect_csharp_DianaGliga11.Model;

namespace Networking.Response
{
    public class NewParticipantResponse : UpdateResponse
    {
        [JsonPropertyName("participant")]
        public Participant Participant{get;set;}

        public NewParticipantResponse()
        {
        }

        public NewParticipantResponse(Participant participant)
        {
            this.Participant = participant;
        }


    }
}

File: Networking/src/main/java/response/NewParticipantResponse.java
package response;

import example.model.Participant;

public class NewParticipantResponse implements UpdateResponse{
    private final Participant participant;

    public NewParticipantResponse(Participant participant) {
        this.participant = participant;
    }

    public Participant getParticipant() {
        return participant;
    }
}
